fast/stages/03-gke-multitenant/_module/main.tf

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gke-project-0" {
+  source          = "../../../../modules/project"
+  billing_account = var.billing_account_id
+  name            = "gke-clusters-0"
+  parent          = var.folder_id
+  prefix          = var.prefix
+  group_iam       = var.group_iam
```

Comment:
There should be a way to grant roles to users too. For instance, Connect Gateway for Google Groups requires having Google Workspace. It might be that the customer only has Cloud Identity.

Reply:
Why should you give roles to users instead of groups? It's an antipattern and we never support it in FAST.

Reply:
Also if they want to create a multi-cluster mesh there are also other APIs that need to be enabled in this project

Reply:
CI has groups :)

Reply:
> Also if they want to create a multi-cluster mesh there are also other APIs that need to be enabled in this project

sure, can you add all the APIs that are needed to `main.tf`?

Reply:
The user access thing is a limitation of connect gateway
Have a look here.
https://cloud.google.com/anthos/multicluster-management/gateway/setup-groups
According to that you need a Google Workspace edition to be albe to enable the LDAP thing. Customers that use CI will not be able to set it up.

Reply:
added a `iam` variable that allows passing users and service accounts for project-level IAM